packages/playground/blueprints/src/lib/steps/run-sql-query.ts

```diff
@@ -0,0 +1,50 @@
+import { StepHandler } from '.';
+
+/**
+ * @inheritDoc runSqlQuery
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ * 	    "step": "runSqlQuery",
+ * 		"path": "wordpress.org/plugins"
```

Comment:
is this the wrong example code?

Reply:
@dmsnell  This file has been removed.

https://github.com/WordPress/wordpress-playground/pull/745/files